src/mono/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -336,7 +336,7 @@ public void SetCustomAttribute(ConstructorInfo con, byte[] binaryAttribute)
             if (res is TypeBuilder)
             {
                 if (throwOnError)
-                    throw new TypeLoadException(string.Format("Could not load type '{0}' from assembly '{1}'", name, this.name));
+                    throw new TypeLoadException(SR.Format("Could not load type '{0}' from assembly '{1}'", name, this.name));
```

Comment:
This should either use `string.Format` or the message should be moved into `Strings.resx`.